bin/qc.py

```diff
@@ -142,9 +151,11 @@ def go(args):
 
 
     qc_line = { 'sample_name' : args.sample,
+                    'count_N' : count_N,
                 'pct_N_bases' : "{:.2f}".format(pct_N_bases),
           'pct_covered_bases' : "{:.2f}".format(pct_covered_bases), 
            'longest_no_N_run' : largest_N_gap,
+             'depth_coverage' : "{:.2f}".format(depth_coverage),
```

Comment:
Round up to next integer for depth of coverage.

Reply:
Done with math.ceil (if thats not ok I can do it with another solution)